src/main.cc

```diff
@@ -351,7 +351,9 @@ int main(int argc, char* argv[]) {
       LOG_DEBUG << "Running " << (*cmd_to_run).first;
       LiteClient client(config);
       ret_val = (*cmd_to_run).second(client, commandline_map);
-      is_reboot_required = client.isRebootRequired();
+      if (cmd == "daemon") {
+        is_reboot_required = client.isRebootRequired();
+      }
```

Comment:
maybe add an "else { you need to reboot now }"?

Reply:
A similar message is yielded before this place https://github.com/foundriesio/aktualizr-lite/blob/ead98c90887c71aa15b4993036b52f671149ce12/src/helpers.cc#L530

Reply:
ah - okay. looks good then. lets ship it